apps/devs-who-run-vue/src/app/core/services/auth.service.ts

```
@@ -0,0 +1,61 @@
+interface RegisterCredentials extends LoginCredentials {
+  email: string;
+  confirmPassword: string;
+}
+
+export class AuthService {
+  private static readonly API_URL = '/api/auth';
+  private static readonly GITHUB_CLIENT_ID = import.meta.env.VITE_GITHUB_CLIENT_ID;
+  private static readonly GITHUB_REDIRECT_URI = `${window.location.origin}/auth/github/callback`;
+
+  static async login(credentials: LoginCredentials): Promise<User> {
+    const response = await axios.post(`${this.API_URL}/login`, credentials);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)